### ISSUE TITLE
[Bug] Panic when closing the window

### ISSUE THREAD

**Issue by Seldom-SE** [aevyrie/bevy_framepace#9]
bevy_framepace panics when closing the game's window. I'm on Windows 10.

```TOML
[dependencies]
bevy = "0.8"
bevy_framepace = "0.5"
```

```Rust
use bevy::prelude::*;
use bevy_framepace::FramepacePlugin;

fn main() {
    App::new()
        .add_plugins(DefaultPlugins)
        .add_plugin(FramepacePlugin::default())
        .run();
}
```

```
PS C:\Users\Hayde\Documents\Development\bevy_test> cargo run
    Finished dev [unoptimized + debuginfo] target(s) in 0.42s
     Running `target\debug\bevy_test.exe`
2022-08-01T23:20:59.485277Z  INFO bevy_render::renderer: AdapterInfo { name: "Radeon RX 570 Series", vendor: 4098, device: 26591, device_type: DiscreteGpu, backend: Vulkan }
2022-08-01T23:20:59.615713Z  INFO bevy_framepace: [Update] Detected refresh rate is: 60 fps
2022-08-01T23:20:59.676518Z  WARN bevy_framepace: [Frame Drop] 73.53ms (+56578.85μs)
2022-08-01T23:20:59.755949Z  WARN bevy_framepace: [Frame Drop] 27.63ms (+10683.25μs)
2022-08-01T23:21:00.918444Z  WARN bevy_framepace: [Frame Drop] 90.74ms (+73786.45μs)
2022-08-01T23:21:00.923103Z  INFO bevy_winit: Skipped event for closed window: WindowId(00000000-0000-0000-0000-000000000000)
2022-08-01T23:21:00.923727Z  INFO bevy_winit: Skipped event for closed window: WindowId(00000000-0000-0000-0000-000000000000)
2022-08-01T23:21:00.934491Z  INFO bevy_winit: Skipped event for closed window: WindowId(00000000-0000-0000-0000-000000000000)
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', C:\Users\Hayde\.cargo\registry\src\github.com-1ecc6299db9ec823\bevy_framepace-0.5.0\src\lib.rs:101:51
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\debug\bevy_test.exe` (exit code: 101)
```

**Comment by aevyrie** [comment]
This has been fixed in #19

**Comment by Seldom-SE** [comment]
Thank you!
